packages/sanity/src/structure/comments/src/components/reactions/CommentReactionsBar.tsx

```diff
@@ -48,29 +55,19 @@ function groupReactionsByName(reactions: CommentReactionItem[]) {
   return sorted as [CommentReactionShortNames, CommentReactionItem[]][]
 }
 
-const ReactionButtonCard = styled(Card)`
-  max-width: max-content;
-  border: 1px solid var(--card-border-color) !important;
-`
-
-const renderMenuButton = ({open}: {open: boolean}) => (
-  <ReactionButtonCard
-    __unstable_focusRing
-    forwardedAs="button"
-    pressed={open}
-    radius={6}
-    tone="transparent"
-    type="button"
-  >
-    <Tooltip animate content="Add reaction">
-      <Flex align="center" justify="center" padding={2} paddingX={3}>
-        <Text muted size={1}>
-          <ReactionIcon />
-        </Text>
+const renderMenuButton = ({open}: {open: boolean}) => {
+  return (
+    <UIButton fontSize={1} mode="ghost" padding={0} radius="full" selected={open}>
```

Comment:
Preferring usage of regular `<Button>` over `<Card>` components here. It does feel a little counter-intuitive having the `<Tooltip>` nested within the button itself, but required as it the `<MenuButton>` won't correctly return focus on close (you already accounted for this!)